src/main/java/uk/gov/hmcts/reform/pip/data/management/models/publication/ListType.java

```diff
@@ -17,7 +17,8 @@ public enum ListType {
     MAGS_STANDARD_LIST(LocationType.VENUE),
     CIVIL_DAILY_CAUSE_LIST(LocationType.VENUE),
     FAMILY_DAILY_CAUSE_LIST(LocationType.VENUE),
-    CIVIL_AND_FAMILY_DAILY_CAUSE_LIST(LocationType.VENUE);
+    CIVIL_AND_FAMILY_DAILY_CAUSE_LIST(LocationType.VENUE),
+    SSCS_DAILY_LIST(LocationType.VENUE);
```

Comment:
Should this be VENUE? (I'm wondering if it's regional instead)

Reply:
Just checked ticket, think it should be region. Updating it.